### PR TITLE
fix: #776

### PR DESCRIPTION
**Pull request by maduvena**
#776

### CODE REVIEW

File: jans-auth-server/server/src/main/java/io/jans/as/server/service/MDSUpdateTimer.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import io.jans.service.timer.schedule.TimerSchedule;
+import org.slf4j.Logger;
+
+/**
+ * @author madhumitas
+ *
+ */
+@ApplicationScoped
+@Named
+public class MDS3UpdateTimer {
```
Comment (yuriyz) [inline jans-auth-server/server/src/main/java/io/jans/as/server/service/MDSUpdateTimer.java]: It's not clear why timer with event is put in jans-auth-server. Shouldn't it be in fido2-server? fido2-server is separate war and can carry it.

File: jans-auth-server/server/src/main/java/io/jans/as/server/service/MDSUpdateTimer.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	@Inject
+	private Logger log;
+
+	@Inject
+	private Event<TimerEvent> timerEvent;
+
+	@Inject
+	private TocService tocService;
+
+	public void initTimer() {
```
Comment (yuriyz) [inline jans-auth-server/server/src/main/java/io/jans/as/server/service/MDSUpdateTimer.java]: Formatting is messed. (Check other java files, e.g. spaces (vs tabs), curly brackets (starting bracket is always in same line))

**Review by yuriyz — CHANGES_REQUESTED** [review]

File: jans-auth-server/server/src/main/java/io/jans/as/server/service/MDSUpdateTimer.java (hunk 1 above)
Comment (maduvena) [reply jans-auth-server/server/src/main/java/io/jans/as/server/service/MDSUpdateTimer.java]: This was in error, corrected

**Review by yuriyz — CHANGES_REQUESTED** [review]
fido2-server app initializer has to be used instead of jans-auth-server

**Review by yuriyz — APPROVED** [review]
Approved since changes are moved out of AS. Now we should let @yurem check fido related changes.

**Review by yurem — APPROVED** [review]

**Pull request merged**